### PR TITLE
Security package added, SecuriOnfiguration.java class create (default…

### PR DESCRIPTION
**Pull request by nobasidi**
… security configuration added), AppPasswordEncoder.java class create with bean function to create an BCryptPasswordEncoder bean

### CODE REVIEW

File: pom.xml
Comment (94Noba) [inline pom.xml:1]: Ok

File: pom.xml [hunk 1]
```diff
@@ -43,6 +43,11 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+	  <dependency>
+      	<groupId>org.springframework.boot</groupId>
```
Comment (94Noba) [inline pom.xml:48]: Add indentation

File: src/main/java/sn/optimizer/Face/security/AppPasswordEncoder.java
Comment (94Noba) [inline src/main/java/sn/optimizer/Face/security/AppPasswordEncoder.java:1]: Nice. OK

File: src/main/java/sn/optimizer/Face/security/SecurityConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@EnableWebSecurity
```
Comment (94Noba) [inline src/main/java/sn/optimizer/Face/security/SecurityConfiguration.java:11]: Not necessery to enable secure method

File: src/main/java/sn/optimizer/Face/security/SecurityConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+@EnableMethodSecurity(prePostEnabled = true, securedEnabled = true)
+public class SecurityConfiguration 
+{
+    @Bean
+    protected SecurityFilterChain securityConfigutation(HttpSecurity http) throws Exception
+    {
```
Comment (94Noba) [inline src/main/java/sn/optimizer/Face/security/SecurityConfiguration.java:17]: csrf is desabled you have to setting session management to staless  very important

Comment (nobasidi) [reply src/main/java/sn/optimizer/Face/security/SecurityConfiguration.java:17]: Ok thank for this addvice

File: src/main/java/sn/optimizer/Face/security/SecurityConfiguration.java (hunk 2 above)
Comment (94Noba) [reply src/main/java/sn/optimizer/Face/security/SecurityConfiguration.java:11]: better to deactivate it and activate it if necessary

Comment (nobasidi) [reply src/main/java/sn/optimizer/Face/security/SecurityConfiguration.java:11]: Sure

**Pull request closed without merging**